RadixWallet/Features/ShieldSetup/SelectFactorSources/Children/SelectFactorSources+View.swift

```diff
@@ -0,0 +1,134 @@
+import ComposableArchitecture
+import SwiftUI
+
+// TODO: move to Sargon
```

Comment:
👍 I really like that you write `// TODO: move to Sargon` like this :) because otherwise you know I will come and review "please move to sargon" :)